runtime/lua/vim/highlight.lua

```diff
@@ -95,7 +99,7 @@ function highlight.on_yank(opts)
   pos1 = {pos1[2] - 1, pos1[3] - 1 + pos1[4]}
   pos2 = {pos2[2] - 1, pos2[3] - 1 + pos2[4]}
 
-  highlight.range(bufnr, yank_ns, higroup, pos1, pos2, event.regtype, event.inclusive, 200)
+  highlight.range(bufnr, yank_ns, higroup, pos1, pos2, { event.regtype, event.inclusive, 200 })
```

Comment:
Shouldn't this be:

```suggestion
  highlight.range(bufnr, yank_ns, higroup, pos1, pos2, { 
    regtype = event.regtype, 
    inclusive = event.inclusive, 
    priority=200 
  })
```

Reply:
Yep, I stopped working on this when mjlbach pushed his PR. It also needs a round of formatting...